apps/ledger-live-mobile/src/components/PostOnboarding/PostOnboardingEntryPointCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Flex, Icons, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { getDeviceModel } from "@ledgerhq/devices/lib/index";
+import styled from "styled-components/native";
+import {
+  useHideWalletEntryPointCallback,
+  usePostOnboardingDeviceModelId,
+  usePostOnboardingEntryPointVisibleOnWallet,
+} from "@ledgerhq/live-common/postOnboarding/hooks/index";
+import { useNavigateToPostOnboardingHubCallback } from "../../logic/postOnboarding/useNavigateToPostOnboardingHubCallback";
+import Touchable from "../Touchable";
+import darkPlaceholderImage from "../../images/illustration/Dark/_000_PLACEHOLDER.png";
+import lightPlaceholderImage from "../../images/illustration/Light/_000_PLACEHOLDER.png";
+
+const PlaceholderImage = styled.Image.attrs(p => ({
+  source:
+    p.theme.colors.type === "dark"
+      ? darkPlaceholderImage
+      : lightPlaceholderImage,
+  resizeMode: "contain",
+}))`
+  height: 100px;
+  width: 100px;
+  margin: 7px;
+`;
+
+const PostOnboardingEntryPointCard: React.FC<Record<string, never>> = () => {
```

Comment:
`Record<string, never>` why do you need this here?

Reply:
typescript now screams when putting `{}` for props and recommends using Record<string, never> in case the component takes no props